code/modules/shuttle/navigation_computer.dm

```diff
@@ -57,6 +60,25 @@
 	else
 		QDEL_NULL(my_port)
 
+/// "Initializes" any default port ids we have, done so add_jumpable_port can be a proper setter
+/obj/machinery/computer/camera_advanced/shuttle_docker/proc/set_init_ports()
+	var/list/init_ports = jump_to_ports.Copy()
+	jump_to_ports = list() //Reset it so we don't get dupes
+	for(var/port_id in init_ports)
+		add_jumpable_port(port_id)
+		
+/obj/machinery/computer/camera_advanced/shuttle_docker/proc/add_jumpable_port(port_id)
+	if(!length(jump_to_ports))
+		actions += new /datum/action/innate/camera_jump/shuttle_docker(src)
+	jump_to_ports[port_id] = TRUE
```

Comment:
How does this action ever get added for mapload shuttle consoles?

Seems like `add_jumpable_port` is only called in `Initialize` from within a `if(!mapload)` conditional

Reply:
See above